src/main/webapp/aboutus.html

```diff
@@ -29,29 +29,31 @@
       </ul>
     </nav>
     <h1>About Our Team</h1>
-    <h2>Teammate A Name</h2>
+    <h2>Jingqi</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz: Excited!!! \^o^/, Apprehensive, Motivated</li>
+      <li>
+        Aspirational Hobby: Learning as many different languages as I can!
+      </li>
+      <li>Ask me About: Singing k with the saddest Chinese songs</li>
     </ul>
     <h2>Teammate B Name</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz:</li>
+      <li>Aspirational Hobby:</li>
+      <li>Ask me About:</li>
     </ul>
     <h2>Teammate C Name</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz:</li>
+      <li>Aspirational Hobby:</li>
+      <li>Ask me About:</li>
     </ul>
     <h2>Teammate D Name</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz:</li>
```

Comment:
Testing leaving comments on a pull request haha.

I'm not sure why there is a space deleted on all the lines in the lists, may be some IDE auto formatting or something. Anyway, I think it may be a good idea to avoid changing code that is likely to be modified by other people because that may lead to merge conflicts down the road.

Reply:
+1 would be great to look into the IDE configuration.

Reply:
Yup okay! Didn't realise there was an auto formatter enabled but I have rectified it for now and will be continuing to explore haha